shadow-dom/focus-navigation/resources/focus-utils.js

```diff
@@ -1,17 +1,27 @@
 'use strict';
 
-function navigateFocusForward() {
-  // TAB = '\ue004'
-  return test_driver.send_keys(document.body, "\ue004");
+function waitForRender() {
+  return new Promise(resolve => requestAnimationFrame(() => requestAnimationFrame(resolve)));
+}
+
+async function navigateFocusForward() {
+  await waitForRender();
+  const kTab = '\uE004';
+  await new test_driver.send_keys(document.documentElement,kTab);
```

Comment:
@dizhang168 @josepharhar Is there a reason of why send_keys's target  gets changed to `document.documentElement` from `document.body`?